lib/matplotlib/__init__.py

```diff
@@ -1476,7 +1476,6 @@ def _jupyter_nbextension_paths():
 default_test_modules = [
     'matplotlib.tests.test_png',
     'matplotlib.tests.test_units',
-    'matplotlib.tests.test_widgets',
```

Comment:
Sorry for being dense, but why does this variable even exist?  Why do we have a default whitelist of tests?